AmazonChimeSDK/AmazonChimeSDKTests/internal/utils/ConcurrentDictionaryTests.swift

```diff
@@ -74,7 +74,7 @@ class ConcurrentDictionaryTests: XCTestCase {
         let mainThreadEndedExpectation = XCTestExpectation(
             description: "The main thread was ended")
 
-        DispatchQueue.global(qos: .background).async {
+        DispatchQueue.global(qos: .userInteractive).async {
```

Comment:
Can you add comment to this change and explain?

Reply:
The quality-of-service (QoS) class has different levels, `userInteractive` has higher priority than `background`, which is performed more quickly and with more resources than lower priority work. And the tests fail due to the short timeout, in order to pass tests, we give them higher priority to perform quickly.

Reply:
Make sense. Could you add comments to the code with your explanation?

Reply:
sure